Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyBoolComboBoxCtrl.hxx

```diff
@@ -25,7 +25,7 @@ class QPushButton;
 
 namespace AzToolsFramework
 {
-    //just a test to see how it would work to pop a dialog
+    using ComboBoxSelectFunc = AZ::Edit::AttributeFunction<bool(bool)>*;
```

Comment:
To be more accurate with the naming please remove the pointer from the type:

````
using ComboBoxSelectFunc = AZ::Edit::AttributeFunction<bool(bool)>;
`````

This represents correctly it's the function type, then all the places where a pointer to the type is used please use `ComboBoxSelectFunc*`.

This is the same done in other places like in `PropertyAssetCtrl.hxx`:

````
        using EditCallbackType = AZ::Edit::AttributeFunction<void(const AZ::Data::AssetId&, const AZ::Data::AssetType&)>;
        using ClearCallbackType = AZ::Edit::AttributeFunction<void()>;

        EditCallbackType* m_editNotifyCallback = nullptr;
        ClearCallbackType* m_clearNotifyCallback = nullptr;
````